src/main/java/com/sonymobile/tools/gerrit/gerritevents/GerritConnection.java

```diff
@@ -94,6 +94,7 @@
     private AuthenticationUpdater authenticationUpdater = null;
     private final Set<ConnectionListener> listeners = new CopyOnWriteArraySet<ConnectionListener>();
     private int sshRxBufferSize = SSH_RX_BUFFER_SIZE;
+    private StringBuilder eventBuffer = null;
```

Comment:
ok, true but as @rsandell pointed out, using the GerritConnection in this way would not only cause problems with the StringBuilder but also with the sshConnection, the Watchdog and so on.
The normal way would be

```
for (int i = 0; i < 3; i++) {
  GerritConnection con = new GerritConnection(...)
  Thread thread = new Thread(con)
  thread.run()
}
```